src/frontend/components/Felleskomponenter/Datovelger/Datovelger.test.tsx

```diff
@@ -50,7 +53,7 @@ describe('Datovelger', () => {
                     tilOgMed,
                 };
             },
-            { wrapper: IntlProvider, initialProps: { locale: 'nb' } }
+            { wrapper: IntlProviderMedLocale }
```

Comment:
API-et for wrapper er endret så man kan ikke sende inn `initialProps` på samme måte som før